docs/cover.md

```diff
@@ -1,57 +1,71 @@
+
 ```
-go tool cover -html=c.out -o coverage.html
+$ go tool cover -html=cover.out
 ```
 
-Please note. At the moment it's not possible to generate coverage report for AWK functions, similar to what is possible for Go:
+To write the HTML file to a specified file instead of launching a web browser, use `-o`:
+
 ```
-go tool cover -func=c.out                # this will not work :(
+$ go tool cover -html=cover.out -o cover.html
 ```
-This won't work because the `go tool cover` functionality for `-func` is only capable of processing the Go code. Luckily, the `-html` report option is generic enough to process any source code! 
 
-If you want to see coverage-annotated source code (this might be useful for debug purposes), just add `-d` flag in addition to `-covermode`:
+If you want to see coverage-annotated source code, use the `-d` option in addition to `-covermode`. This might be useful for debugging, or to see how GoAWK's coverage support works under the hood:
+
 ```
-goawk -f a.awk -covermode set -d
+$ goawk -f prog.awk -covermode set -d
+BEGIN {
+    __COVER["3"] = 1
+    print "will always run"
+    if ((1 + 1) == 2) {
+        __COVER["1"] = 1
+        print "should run"
+    } else {
+        __COVER["2"] = 1
+        print "won't run"
+    }
+}
 ```
 
 
-## CLI options for coverage
+## All command-line options
+
+- `-coverprofile fn`: set the coverage report filename to `fn`. If this option is specified but `-covermode` is not, the coverage mode defaults to `set`.
+- `-covermode mode`: set the coverage mode to `mode`, which can be one of:
+  - `set`: did each statement run?
+  - `count`: how many times did each statement run? (produces a heat map report)
+- `-coverappend`: append to coverage profile instead of overwriting it. This allows you to accumulate coverage data across several different runs of the program.
 
-- `-coverprofile <file>`
-  - sets the cover report filename to put collected coverage profile data.
-- `-covermode <mode>`
-  - `mode` can be one of:
-    - `set` - did each statement run?
-    - `count` - how many times did each statement run? (Produces heat maps report).
-- `-coverappend`
-  - if this option is provided the profile data will append to existing, otherwise the profile file will be first truncated.
 
 ## Future work
 
-- Add a way to support per-function coverage report similar to `go tool cover -func=c.out`
-- More complete handling for coverage cases for `if/else`. That is not only we want to check if `if` body was visited but also we want to make sure the case where `if` body was *not* visited is also covered. In other words we want to make sure that both variants of `if` condition is covered. 
+- Add a way to support per-function coverage reporting, similar to `go tool cover -func=cover.out`.
+- More complete handling for coverage of `if`/`else`. Not only do we want to check if the `if` body was visited, but we also want to handle the case where the `if` body was *not* visited.
```

Comment:
I don't know exactly what this means. It seems to me the coverage feature already supports if/else?

Reply:
Hey, here I was talking of the scenario, mentioned here: https://github.com/benhoyt/goawk/pull/154#discussion_r996465307

Reply:
If need be we can adjust the wording to:

- More complete handling for coverage of `if`/`else` (see details: https://github.com/benhoyt/goawk/pull/154#discussion_r996465307).